README.md

```diff
@@ -15,13 +15,13 @@ This library is published for Scala 2.12.x, 2.13.x and 3.1.x:
 
 ```scala
 // sbt
-"org.typelevel" %% "scalac-options" % "0.1.3"
+"org.typelevel" %% "scalac-options" % "0.1.7"
 
 // mill
-ivy"org.typelevel::scalac-options:0.1.3"
+ivy"org.typelevel::scalac-options:0.1.7"
 
 // Scala CLI
-//> using lib "org.typelevel::scalac-options:0.1.3"
+//> using dep "org.typelevel::scalac-options:0.1.7"
```

Comment:
nitpick: the quotes aren't necessary anymore (I think they were in some very old Scala-CLI versions, but it's been a while)

Reply:
Well spotted, fixed.